documentation/docs/reference/environment.md

```diff
@@ -10,7 +10,9 @@ keywords:
 
 Currently Chronicle supports the following environment variables:
 
-* `CONFIG_PATH="<FILE_PATH>"`: sets the file path to the `config.toml` file;
-* `INX=<true|false>`: enables/disables INX;
-* `API=<true|false>`: enables/disables the REST API;
-* `METRICS=<true|false>`: enables/disables the Metrics Server;
+* `MONGODB_CONN_STR`: sets the MongoDb connection string including credentials;
+* `INFLUXDB_USERNAME`: sets the InfluxDb user;
+* `INFLUXDB_PASSWORD`: sets the InfluxDb password;
+* `JWT_IDENTITY`: sets the filepath to a JWT identity file;
+* `JWT_PASSWORD`: sets the JWT password;
+* `JWT_SALT`: sets the JWT salt;
```

Comment:
There are a few more now, like `INX_URL`.

Reply:
yeah, sorry ... I branched off of `main`.

Reply:
Right, I forgot. Well once we merge this we should create a new PR on the feature branch to add the new stuff.